server/routes/club.js

```diff
@@ -0,0 +1,19 @@
+import express from "express";
+import { createClub, deleteClub, getClub, getAllClubs } from "../controllers/club.js";
+import auth from "../middleware/auth.js";
+
+const router = express.Router();
+
+//addClub
+router.post("/createClub", auth, createClub);
```

Comment:
upload.single("image") middleware has to be used for handling image uploads in create club request